docs/user/security/audit-logging.asciidoc

```diff
@@ -313,7 +313,11 @@ Possible values:
 `deletion`
 
 | [[field-event-outcome]] `event.outcome`
-| Denotes whether the event represents a success or failure. 
+a| Denotes whether the event represents a success or failure: 
+
+* Any actions that the user is not authorized to perform are logged with outcome:  `failure`
+* Authorized read operations are only logged after successfully fetching the data from {es} with outcome: `success`
+* Authorized create, update, or delete operations are logged before attempting the operation in {es} with outcome: `unknown`
 
 Possible values:
```

Comment:
Not sure we need the line `Possible values: success, failure, unknown` as this is now explained in the bullet points.

Reply:
I agree, it's probably not necessary but think it's still useful for consistency since all the other settings in this doc have a "possible values" section.